src/components/AppSettingsMenu.vue

```diff
@@ -198,6 +216,19 @@ export default {
 					this.loadingAvatarSettings = false
 				})
 		},
+		onToggleSearchPriorityBody(e) {
+			this.loadingPrioritySettings = true
+
+			this.$store
+				.dispatch('savePreference', {
+					key: 'search-priority-body',
+					value: e.target.checked ? 'true' : 'false',
+				})
+				.catch((error) => Logger.error('could not save preferences', { error }))
+				.then(() => {
+					this.loadingPrioritySettings = false
+				})
```

Comment:
my typical requested review: please make the method async and use the await syntax with a try-catch :)

Reply:
I should start remembering that